Assets/Scripts/Characters/Titan/Attacks/GrabAttack.cs

```diff
@@ -8,7 +8,9 @@ public GrabAttack()
         {
             BodyParts = new[] {BodyPart.HandLeft, BodyPart.HandRight};
         }
-        private string AttackAnimation { get; set; }
+
+        // HACK: It is Empty in certain situations. This fixes that.
```

Comment:
Let's find out what issue should fix this.

Reply:
Create a new bug with reproduction steps.